Microsoft.Packaging.Tools/docs/trimming.md

```diff
@@ -45,21 +83,21 @@ By default the application is a *root*, as well as all `PackageReference`s from
 
 The direct packages references may be excluded from the set of *roots* by specifying the property `RootPackageReference=false`.
 
-Additional file *roots* may be specified using the `TrimFilesRootFiles` item.
+Additional file *roRootPackageReferenceots* may be specified using the `TrimFilesRootFiles` item.
 Additional package *roots* may be specified using the `TrimFilesRootPackages` item.
 
 ### Trimmable
-Files or packages may be treated as *trimmable*.  Essentially this means that when the file or package is encountered during graph traversal, traversal will halt down that path.
+Files or packages may be treated as *trimmable*.  Essentially this means that when the file or package is encountered while examining dRootPackageReferenceependencies, that file or package will not be included nor will its dependencies unless otherwise referenced.
```

Comment:
Thanks looks like these were accidentally introduced by my markdown editor when I was trying to do searches.  I'll clean them up.